src/main/java/ru/sbtqa/tag/parsers/TestPullRequest.java

```diff
@@ -0,0 +1,12 @@
+package ru.sbtqa.tag.parsers;
+
+public class TestPullRequest {
+    
+    public void thrower() {
+        try {
+            int i = 19;
+        } catch (Exception e) {
+            System.out.println("blep");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)